music21/metadata/__init__.py

```diff
@@ -1773,7 +1786,7 @@ class RichMetadata(Metadata):
     'scenarist', 'sceneNumber', 'scientificAdvisor', 'screenplayAuthor', 'scribe',
     'sculptor', 'secretary', 'setDesigner', 'singer', 'source', 'sourcePath',
     'spatialCoverage', 'speaker', 'standardsBody', 'storyteller', 'subject',
-    'subtitle', 'surveyor', 'suspectedComposer', 'tableOfContents', 'teacher',
+    'surveyor', 'suspectedComposer', 'tableOfContents', 'teacher',
```

Comment:
'surveyor' can go also.  Probably 'secretary' 'landscapeArchitect' 'meetingOrganizer' 'architect' 'mediator'.

If in doubt leave it in ('illuminator' is actually helpful for me!) but 'cartographer'...

Reply:
Think of it this way, if it's about as absurd as adding '.calorieCount' or '.horsePower' it probably shouldn't be in music21

Reply:
Heh.  I would have removed 'illuminator' under that way of thinking... I am not going to be the expert on what is useful, so I will count on you to tell me what to remove/add in the way of supported metadata properties.

Reply:
Take your best guess and I'll let know if there's anything that's too far.

Reply:
The point here is that it's too hard to see what's in the richMetadata now, because too much is the same.  Maybe the test should be to show what's in RichMD and not in MD...

Reply:
Done.